### PR TITLE
Fixed nolana

### PR DESCRIPTION
**Pull request by TetianaParanich**

### CODE REVIEW

**Review by MaksimTerekhov — APPROVED** [review]
please fix linter issues

File: cypress/e2e/data-import/filter-and-delete-import-logs-on-view-all.cy.js [hunk 1]
```diff
@@ -20,23 +21,26 @@ describe('ui-data-import: A user can filter and delete import logs from the "Vie
   completedDate.setDate(completedDate.getDate() + 1);
   let firstUser;
```
Comment (IhorBohdan) [inline cypress/e2e/data-import/filter-and-delete-import-logs-on-view-all.cy.js:22]: fix linter issues

File: cypress/e2e/data-import/filter-and-delete-import-logs-on-view-all.cy.js [hunk 2]
```diff
@@ -69,21 +73,21 @@ describe('ui-data-import: A user can filter and delete import logs from the "Vie
       });
   });
 
-  after(() => {
-    Users.deleteViaApi(firstUser.userId);
-    Users.deleteViaApi(secondUser.userId);
-    // TODO delete all created instances and holdings
-  });
+  // after(() => {
```
Comment (IhorBohdan) [inline cypress/e2e/data-import/filter-and-delete-import-logs-on-view-all.cy.js:76]: Does need here code in comments?

File: cypress/e2e/data-import/filter-and-delete-import-logs-on-view-all.cy.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/filter-and-delete-import-logs-on-view-all.cy.js:22]: Done

File: cypress/e2e/data-import/filter-and-delete-import-logs-on-view-all.cy.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/filter-and-delete-import-logs-on-view-all.cy.js:76]: Done

**Pull request merged**